example/flutter_rest_data_example/lib/data/beer_model.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter_rest_data/flutter_rest_data.dart';
+import 'package:flutter_rest_data_example/data/ingredient_model.dart';
+
+class BeerModel extends JsonApiModel {
+  BeerModel(JsonApiDocument jsonApiDoc) : super(jsonApiDoc);
+
+  String get name => attributes!['name'];
+
```

Comment:
@algodave 
I don't like this cast here, but `attributes` are nullable in the api at the moment. I'd rather make this property an empty map, if there is no attributes, but maybe that's just me.

Reply:
Good catch! We definitely need to make `attributes` non nullable.